src/test/java/kg/jarkyn/server/FileHelpers.java

```diff
@@ -0,0 +1,39 @@
+package kg.jarkyn.server;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileHelpers {
+    private static String projectFolderPath = System.getProperty("user.dir");
+    public static String publicFolderPath = projectFolderPath + "/src/test/resources/public";
+    public static String publicFolderRequestPath = "/";
+    static String plainFileName = "file1";
```

Comment:
I actually thought this is a convention for package-private, which is what I needed here as this is only used for tests
(will most likely have to make it public once I group code into related packages)